### DISCUSSION TITLE
JSON Schema Visualizer?

### DISCUSSION THREAD

**Discussion by vscaemh** [json-schema-org/community#844] [Q&A]
Any ideas on a good json schema viewer? I like https://github.com/jlblcc/json-schema-viewer - demo at https://jlblcc.github.io/json-schema-viewer but it has not been updated since 7 years ago.

**Comment by jviotti** [comment]
@vscaemh Are you mainly looking for a tree view? What are the features you would love from a JSON Schema viewer/explorer?

**Reply by vscaemh** [reply]
For a higher-level perspective I am looking for an approach to facilitate data contract negotiations.
I find that JSON schema can describe the contract, but the process of getting to the contract is difficult.

So I am thinking of the holistic view of utilizing JSON schema - also the elements surrounding the actual schema, i.e. a viewer and something like https://github.com/[bcwaldon/warlock](https://github.com/bcwaldon/warlock)

JSON Schema is not a good way to discuss with shareholders, as the schema files are cumbersome to use in a discussion.
I have gotten much further with visualizations.

I am very sorry for the delay in my response, I must have missed it somehow.
